docs/playback_filters.md

```diff
@@ -2,186 +2,259 @@
 
 The QCTools preview window is intended as an analytical playback environment that allows the user to review video through multiple filters simultaneously. Often inconsistencies within the graph layout may be better understood by examining those frames in the playback window. The playback window includes two viewing windows which may be set to different combinations of filters. This allows a user to playback a video by multiple forms of analysis simultaneously just as viewing a waveform and a vectorscope side-by-side or seeing the video with highlighted pixels that are outside of broadcast range while seeing the waveform display.
 
-## Help
+---
+
+## Table of Contents
+
+| A-C          | D-S       | T-Z         |
+| -------------| ---------- | ---------- |
+| [Audio Bit Scope](#audio-bit-scope) | [Datascope](#datascope) | [Temporal Difference](#temporal-difference) |
+| [Audio Frequency](#audio-frequency) | [Extract Planes Equalized](#extract-planes-equalized) | [Temporal Outlier Pixels](#temporal-outlier-pixels) |
+| [Audio Phase Meter](#audio-phase-meter) | [Extract Planes UV Equalized](#extract-planes-uv-equalized) | [Value Highlight](#value-highlight) |
+| [Audio Waveform](#audio-waveform)| [Field Difference](#field-difference) | [Vectorscope](#vectorscope) |
+| [Audio Spectrum](#audio-spectrum) | [Frame Tiles](#frame-tiles) | [Vertical Line Repetitions](#vertical-line-repetitions) |
+| [Audio Vectorscope](#audio-vectorscope) | [Help](#help) | [Vertical Repetition Pixels](#vertical-repetition-pixels) |
+| [Audio Volume](#audio-volume) | [Histogram](#histogram) | [Vectorscope Target](#vectorscope-target) |
+| [Audio Waveform](#audio-waveform) | [Line Select](#line-select) | [Waveform](#waveform) |
+| [Bit Plane](#bit-plane) | [No Display](#no-display) | [Waveform / Vectorscope](#waveform-vectorscope) |
+| [Bit Plane 10 slices](#bit-plane-10-slices) | [Normal](#normal) | [Zoom](#zoom) |
+| [Bit Plane Noise](#bit-plane-noise) | [Pixel Offset Subtraction](#pixel-offset-subtraction) |
+| [Broadcast Illegal Focus](#broadcast-illegal-focus) | [Sample Range](#sample-range) |
+| [Broadcast Range Pixels](#broadcast-range-pixels) | [Saturation Highlight](#saturation-highlight) |
+| [Chroma Adjust](#chroma-adjust) |
+| [CIE Scope](#cie-scope) |
+| [Color Matrix](#color-matrix) |
+
+
+---
+
+## [Audio Bit Scope](#audio-bit-scope)
 
-This selection will launch the Playback Filters window where you may review the documentation.
+Shows an audio bit scope visualization of the audio. See FFmpeg's [abitscope](https://ffmpeg.org/ffmpeg-filters.html#abitscope) filter.
+![Audio Bit Scope](media/playbackfilter_audio_bit_scope.jpg)
```

Comment:
For some reason, images are not loading upon build. @dericed reported a similar problem so it may have to do with the building process, as the code here (and elsewhere) looks right to me.